homeassistant/components/text/device_action.py

```diff
@@ -0,0 +1,80 @@
+"""Provides device actions for Text."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    CONF_DEVICE_ID,
+    CONF_DOMAIN,
+    CONF_ENTITY_ID,
+    CONF_TYPE,
+)
+from homeassistant.core import Context, HomeAssistant
+from homeassistant.helpers import entity_registry
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import ConfigType, TemplateVarsType
+
+from .const import ATTR_VALUE, DOMAIN, SERVICE_SET_VALUE
+
+ATYP_SET_VALUE = "set_value"
+
+ACTION_SCHEMA = cv.DEVICE_ACTION_BASE_SCHEMA.extend(
+    {
+        vol.Required(CONF_TYPE): ATYP_SET_VALUE,
```

Comment:
Are we missing the action type in the translation strings? See eg switch:

https://github.com/home-assistant/core/blob/1908feab7964a3810b6210b58bb869eaadb6b68d/homeassistant/components/switch/strings.json#L4-L8